samples/csharp_dotnetcore/51.teams-messaging-extensions-action/Controllers/HomeController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TeamsMessagingExtensionsAction.Controllers
+{
+    public class HomeController : Controller
```

Comment:
Is this controller necessary?   

Can we just use Razor Pages for this?

Reply:
@EricDahlvang , We are using the 2 commands HTML Page & MVC WebView, HTML is Razor page which doesn't use any controller (We are rendering it as razor page itself )method but We have integrated this controller for MVC webview